### DISCUSSION TITLE
GoodCheapFast - Ödev

### DISCUSSION THREAD

**Discussion by FutureBaseEditor** [FutureBaseEditor/FormAppOYU#1] [General]
![good-cheap-fast](https://user-images.githubusercontent.com/128919303/236317980-1cf16e60-9786-4d80-b73c-ede6aa39919f.gif)
![FastCheapGood2](https://user-images.githubusercontent.com/128919303/236317975-ef69c8d6-24f8-4cc3-b7ec-5d297af48195.gif)

**Comment by DilaZerenYilmaz** [comment]
https://github.com/DilaZerenYilmaz/Good-Cheap-Fast

**Reply by FutureBaseEditor** [reply]
No checkbox. Yes button. 
Doğru olan checkbox ama kodlaması kolay oluyor. Biz butonla yapmak istiyoruz

**Reply by DilaZerenYilmaz** [reply]
Buyurunuz: 
https://github.com/DilaZerenYilmaz/Good-Cheap-Fast-Buttons
